src/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Set the project name and version
+project(osconfig-fuzzer VERSION 1.0)
+
+# Specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+find_package(LLVM REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=fuzzer")
+
+add_executable(osconfig-fuzzer target.cpp)
```

Comment:
>target.cpp [](http://example.com/codeflow?start=31&length=10)

Can we do the target in C, like the fuzzed code is?

Reply:
Why would we do that? This is testing code and we benefit from some goodness of the language like STL containers.

Reply:
Do we need STL containers and more such complex types for the fuzzing test instrumentation? Maybe I could see some in a next commit as an example of your reply and understand better.

Reply:
For simplicity and to get this project up quick, I decided to go with C++ which is fine to be used in testing code. I use standard library primitives and feel more confident and robust using them. I don't see a reason for going with plain C here and there is no such requirement that I would be aware. Therefore, I don't intend to use C here.